binder/postBuild

```diff
@@ -31,7 +31,7 @@ chmod +x xstartup
 # Desktop applications
 # Fiji and OMERO insight
 
-OMERO_INSIGHT_VERSION=5.5.11
+OMERO_INSIGHT_VERSION=5.5.14
 OMERO_INSIGHT_URL_BASE=https://github.com/ome/omero-insight/releases/download/v$OMERO_INSIGHT_VERSION
```

Comment:
Exactly the same problem as in https://github.com/ome/omero-guide-fiji/pull/14#discussion_r501630112 - but this time, the travis is pasing.... Not sure why 

cc @jburel

Reply:
As explained in https://github.com/ome/omero-guide-fiji/pull/14#discussion_r501639159, this actually works fine. Thanks